PokeDemoTests/Resources/PokemonList.json

```diff
@@ -0,0 +1,47 @@
+
```

Comment:
Is it required now?

Reply:
Covered in 37dab3f9f4885f47ac8e5f7d2809f2b5aeeb6fb9